add-domain

```diff
@@ -522,3 +522,15 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+www.roblox.com.es
+newappmail.discoverynevers.com
+app-uniswap.org-vote-eth-swap.site
+app-uniswap.org-claim-eth-liquidty.site
```

Comment:
Seems dead to me from both manual lookup and @pyfunceble

![image](https://user-images.githubusercontent.com/44526987/126141021-3efc7be7-28b5-4fb7-aa77-57d0428e1146.png)